### DISCUSSION TITLE
unable to decode video track: invalid FU-A packet (non-starting)

### DISCUSSION THREAD

**Discussion by daburton263** [bluenviron/mediamtx#778] [Questions]
[rtsp simple server.txt](https://github.com/aler9/rtsp-simple-server/files/7892451/rtsp.simple.server.txt)

When I publish a stream from OBS to the rtsp-simple-server and then use ffmpeg to open that stream it works for a while but then I get errors reported by ffmpeg and warnings from the server. I'm running on Windows 10. OBS is running on a different box/network from the server and the ffmpeg is on another box/network. They are all behind NAT with port 1935 open to the server. Anyone know what is going on? The stream continues and I can attach other VLC clients. But the file created by ffmpeg has the video and audio out of sync when I get deeper into the file. When opening a VLC client the audio/video are in sync. A sample of the errors are shown in the attached file.

**Comment by aler9** [comment]
Hello, this happens because there isn't enough bandwidth between the publisher and the server. Decrease the bitrate.

**Comment by daburton263** [comment]
Thanks for your reply. I suspected that might be the case because it didn’t happen on the local network but I was puzzled because my remote location has a 20meg up/down fiber based connection and the server is on a 50 meg up/down. The bitrate I was experimenting with was in the 3-8 meg range. I have never had a problem with an 8 meg RTMP CBR connection to the boxcast servers both up and down. I have been experimenting with an srt server and it seems work without the issues I was seeing with the rstp-simple-server.

[a comment by an automated account is not shown]
